test/system_SUITE.erl

```diff
@@ -0,0 +1,446 @@
+-module(system_SUITE).
+-compile([export_all]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("amqp_client/include/amqp_client.hrl").
+
+-define(SERVER_REJECT_CLIENT, {tls_alert,"unknown ca"}).
+
+all() ->
+    [
+     library,
+     invasive_SSL_option_change,
+     validation_success_for_AMQP_client,
+     validation_failure_for_AMQP_client,
+     whitelisted_certificate_accepted_from_AMQP_client_regardless_of_validation_to_root,
+     removed_certificate_denied_from_AMQP_client,
+     installed_certificate_accepted_from_AMQP_client,
+     whitelist_directory_DELTA,
+     replaced_whitelisted_certificate_should_be_accepted,
+     ensure_configuration_using_binary_strings_is_handled
```

Comment:
Can you put all your tests under a group? Without grouping, I have no idea how to run a specific test, at least with the interface provided by Erlang.mk. Usually in this case, I name the group `non_parallel_tests` by lack of better name.

With this in place, you can run the whole group:
```
make ct-system t=non_parallel_tests
```

Or a single testcase:
```
make ct-system t=non_parallel_tests:library
```

There is probably a name for the top-level "group" but I never found it (though I never looked at the source code either).